tools/travis.sh

```diff
@@ -6,7 +6,7 @@
 
 set -e
 
-IMG='ocaml/opam2:debian-9-ocaml-4.07'
+IMG='ocaml/opam2:debian-unstable'
```

Comment:
I think we'll need to add an `opam switch 4.07` below, the default for this image seems to be 4.06

Reply:
When I watched this build, it used 4.07

Reply:
I've added it.